internal/rules/aws/ec2/no_default_vpc.go

```diff
@@ -31,10 +31,13 @@ var CheckNoDefaultVpc = rules.Register(
 		Severity: severity.High,
 	},
 	func(s *state.State) (results scan.Results) {
-		for _, def := range s.AWS.EC2.DefaultVPCs {
+		for _, vpc := range s.AWS.EC2.VPCs {
```

Comment:
Updated the existing Rule to now check if the `IsDefault` is `true` - previously it was only `DefaultVPCs` so they were all inherently `default`